src/model/immutable/ContentBlockNode.js

```diff
@@ -76,8 +76,7 @@ const decorateCharacterList = (
   return config;
 };
 
-class ContentBlockNode extends (Record(defaultRecord): any)
-  implements BlockNode {
+class ContentBlockNode extends Record(defaultRecord) implements BlockNode {
```

Comment:
Unfortunately the coercion to `any` is required for internal Facebook Flow typechecking on the www repo, so it has to stay like that for now 😞

Reply:
I see. Got it :)